rustls/src/sign.rs

```diff
@@ -397,3 +399,7 @@ pub fn supported_sign_tls13() -> &'static [SignatureScheme] {
         SignatureScheme::ED25519,
     ]
 }
+
+/// Errors while signing
+#[derive(Debug)]
+pub struct SignError;
```

Comment:
Is this supposed to be publicly-constructible? I would expect it to be `pub struct SignError(());` or `pub struct SignError(pub(crate) ());`.

Reply:
Fixed.